docs/_content/standaard/verzoeken/index.md

```diff
@@ -0,0 +1,69 @@
+
+Met verzoeken introduceren we een nieuw concept in het zaakgericht werken. Voor het bepalen van de functionaliteit die een eerste implementatie van de Verzoeken API moet bieden, gaan we uit van de ingediende user stories en andere/eerdere implementaties van het begrip 'verzoeken' (zie de bronnenlijst onderaan). Op basis daarvan kan een eerste set ontwerpbeslissingen en uitgangspunten worden opgesteld. Die zijn in dit document beschreven. Tegelijkertijd zijn over de precieze verhouding tussen 'verzoeken' en een aantal andere concepten in het zaakgericht werken, met name 'zaak' en '(klant)contactmoment', nog een aantal vragen te beantwoorden (zie relaties/kardinaliteiten hieronder). Mogelijk wordt de Verzoeken API naar aanleiding daarvan in de toekomst uitgebreid. 
+
+# Redenen voor het ontwikkelen van een Verzoeken API
+Het (her)introduceren van verzoeken in het zaakgericht werken biedt een aantal voordelen, onder andere op de volgende gebieden:
+* Compleetheid: we standaardiseren een extra deel van het proces dat leidt tot levering van producten en diensten, namelijk (het generieke deel van) de gegevens die nodig zijn voor het aanvragen hiervan. Hiermee kunnen we (het generieke deel van) StUF-EF vervangen.
+* Herkenbaarheid: de klant communiceert met de gemeente niet langer over zaken (een voor hem niet altijd herkenbaar concept), maar over verzoeken, die (als verzoeken correct worden gebruikt) een voor de klant herkenbare (combinatie van) product(en) vertegenwoordigen.
+* Flexibeler (1): gemeenten worden flexibeler in het aanbieden van meerdere gerelateerde producten tegelijk (één verzoek kan leiden tot meerdere zaken). Tegelijkertijd kunnen meerdere verzoeken worden afgehandeld in één zaak.
+* Flexibeler (2): bij het indienen van een verzoek hoeft nog niet duidelijk te zijn hoe dat verzoek zal worden behandeld. In de huidige praktijk moet een vraag om levering van een product of dienst immers direct te worden omgezet naar een zaak van een bepaald zaaktype, anders kan immers geen identificerend kenmerk worden teruggestuurd aan de klant. Door verzoeken onderdeel te maken van het zaakgericht werken, kan het verzoeknummer aan de klant gecommuniceerd (‘uw verzoek is ontvangen!’), zonder dat daarmee is vastgelegd op welke manier de behandeling plaatsvindt (en wellicht is in sommige gevallen een zaak/ontvangstbevestiging niet nodig omdat een verzoek direct geautomatiseerd kan worden afgehandeld). Dit betekent dus ook dat een verzoek op een niet-zaakgerichte manier kan worden afgehandeld!
+* Zuiverheid: er ontstaat een duidelijk onderscheid tussen 'klantorders' en interne 'productie-/pickorders'.
+* Eenvoud: het onderscheid tussen hoofd- en deelzaken kan op termijn vervallen, en wordt vervangen door onderscheid tussen verzoeken en zaken. Uitzoeken hoe het zit met regiezaken.
```

Comment:
> Eenvoud: het onderscheid tussen hoofd- en deelzaken kan op termijn vervallen het in behandeling nemen van een verzoek als zaak

Begrijp niet waarom het onderscheid tussen hoofd- en deelzaken kan vervallen. Als een verzoek heeft geleid tot meerdere zaken dan kan één van die zaken altijd nog dusdanig complex zijn dat een ambtenaar het wil opsplitsen in twee of meer deelzaken. Dat is een interne keuze.

Reply:
Om het niet overdreven complex te maken, kan ik me dan voorstellen dat in dergelijke gevallen in plaats te kiezen voor een onderscheid tussen hoofd- en deelzaken, het verzoek in kleinere zaakjes wordt geknipt.